cordova-lib/spec-plugman/platforms/windows.spec.js

```diff
@@ -192,34 +203,39 @@ beforeEach(function () {
 
                 projectsAddedToSpies.forEach(function (spy) {
                     expect(spy).toHaveBeenCalled();
+                    spy.calls.reset();
```

Comment:
I took out all of the resets.. everything still seems to be working when I test the file individually and also when I run npm test.